pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>token-macro</artifactId>
       <version>1.10</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.docker-java</groupId>
+      <artifactId>docker-java</artifactId>
+      <version>1.3.0</version>
```

Comment:
Only if you can convince docker-plugin to take the same dependency. If you cannot then you need to use the wrapper plugin as a dependency in docker-commons and docker-plugin.

I think a simple wrapper plugin that just wraps the jar file is the best option. When you mix in with other code it starts getting complex.